src/components/Thought.jsx

```diff
@@ -0,0 +1,31 @@
+import moment from "moment";
```

Comment:
What is moment? What does it do? 
Waaaait, is this showing how long ago the user wrote a thought?
I need to know more. How did you get it to show hours and days?

Reply:
Yes, it'a library that's takes care of it for you! But it's no longer maintained so we need a new one.

It's the "fromNow" method :)

```
  const handleDate = date => {
    const newDate = new Date(date);
    return moment(newDate).fromNow();
  }
```;